implementations/micrometer-registry-prometheus/build.gradle

```diff
@@ -4,4 +4,6 @@ dependencies {
     api 'io.prometheus:simpleclient_common'
 
     testImplementation project(':micrometer-test')
+	
+	api 'io.opentelemetry:opentelemetry-api:1.11.0'
```

Comment:
This is not needed, see below.

Reply:
Hellà @jonatan-ivanov. See also my answer on your other comment.